python_modules/dagstermill/dagstermill/examples/repository.py

```diff
@@ -118,6 +120,46 @@ def define_test_notebook_dag_pipeline():
     )
 
 
+DataFrame = as_dagster_type(pd.DataFrame)
+
+
+@solid_definition
```

Comment:
I think I originally wrote ``solid_definition`` but I'm not sure we should use it